<launch>
  <arg name="destination_ip" default="192.168.255.255" doc="The ip address of the camera/s to be triggered" />
  <arg name="destination_interface" default="enp0s31f6" doc="The network interface to broadcast the trigger signal" />
  <arg name="trigger_src" default="timer" doc="When to send action commands. Use 'timer' or 'subscriber'" />
  <arg name="timer_period" default="0.2" doc="Period of timer for sending action commands (s)" />
  <arg name="action_device_key" default="1" doc="The device key for the sent action" />
  <arg name="action_group_key" default="1" doc="The group key for the sent actions" />
  <arg name="action_group_mask" default="1" doc="The group mask for the sent actions" />

  <node name="trigger_node" pkg="avt_vimba_camera" type="trigger_node" output="screen">
    <param name="destination_ip" value="$(arg destination_ip)" />
    <param name="destination_interface" value="$(arg destination_interface)" />
    <param name="trigger_src" value="$(arg trigger_src)" />
    <param name="timer_period" value="$(arg timer_period)" />
    <param name="action_device_key" value="$(arg action_device_key)" />
    <param name="action_group_key" value="$(arg action_group_key)" />
    <param name="action_group_mask" value="$(arg action_group_mask)" />
  </node>
</launch>
